utils/firestore.js

```diff
@@ -3341,6 +3340,25 @@ const getAppSettings = async (appName, selectedParamsArray) => {
     }
 }
 
+/**
+ * Update Notify message delivery status to Firestore.
+ * @param {Object} data 
+ */
+const updateNotifySmsRecord = async (data) => {
+  const snapshot = await db
+    .collection("notifications")
+    .where("phone", "==", data.phone)
```

Comment:
Is there any case where phone number will be duplicated?

Reply:
No. Phone numbers are de-duplicated, so that,  in an API call of bulk messages, each phone number is unique.